packages/commons/src/events/state/utils.ts

```diff
@@ -93,7 +93,8 @@ const updateActions = ActionTypes.extract([
   ActionType.REJECT,
   ActionType.ARCHIVE,
   ActionType.PRINT_CERTIFICATE,
-  ActionType.REQUEST_CORRECTION
+  ActionType.REQUEST_CORRECTION,
+  ActionType.CORRECT
```

Comment:
I'm trying to figure out if there would be a way to do this without actually defining a new `ActionType.CORRECT` at all, just using the the `ActionType.REQUEST_CORRECTION`, while displaying different texts on the UI for the user and doing a different tRPC API request depending on scope of the user.

What are your thoughts?

Reply:
You are absolutely right, all of these can be achieved with REQUEST_CORRECTION action type. The difference between direct correction and correction request can be a cosmetic thing in the frontend using SCOPES.